### ISSUE TITLE
Unable to use link in the confirm email

### ISSUE THREAD

**Issue by caleywoods** [hotsh/rstat.us#462]
So in the process of trying to duplicate @markzalar #461 I discovered I didn't have an email address on my account. I edited my profile and added the account and got the flash notice that a confirmation email had been sent to the email provided.

I checked my email and had the confirmation email from steve+rstatus. The link in the email to confirm my email address when clicked flashes a notice on the page that "can't find user account for this link".

![err0r](http://i.imgur.com/Jv0Mf.png)

So after double checking the link, the link in the email points to rstat.us/confirm_email/true. I would expect this to have a key in the   link to uniquely identify my confirmation, similar to the forgot password link.

So long story short, I ended up unlinking my twitter account and now I'm locked out of my account. The reset password link does come to my email address but when clicking the link I get the "that link is expired, please request a new one" flash message.

So I'm not unable to troubleshoot using my account, I wonder if it's a bug with older accounts.

An update: I tried adding my email address to my other account and the confirm link is rstat.us/confirm_email/false
